### PR TITLE
[4.0][DRAFT] Begin allowing for other icon families

### PR DESCRIPTION
**Pull request by N6REJ**
Pull Request for Issue # .

### Summary of Changes
Allows for custom icon prefix and fixed width

### Testing Instructions
inspect icons.  Verify they display same as before.

### Documentation Changes Required
explain how to use icon layout

### CODE REVIEW

File: layouts/joomla/icon/iconclass.php [hunk 1]
```diff
@@ -13,6 +13,12 @@
 $icon = $displayData['icon'];
 $html = $displayData['html'] ?? true;
 
+// Allow for custom icon family & prefix
+$iconPrefix = $displayData['prefix'] ?? 'fas fa-';
+
+// Set to wide with if true.
```
Comment (brianteeman) [inline layouts/joomla/icon/iconclass.php:19]:
```suggestion
// Set to wide width if true.
```

**Comment by brianteeman** [comment]
fw stands for "fixed width" not "wide"
https://fontawesome.com/how-to-use/on-the-web/styling/fixed-width-icons

**Comment by hans2103** [comment]
@N6REJ smart
But I see two things:

- when `$icon === $icon` and `HTML = false` the `wide` will not be set. 
- you are setting `iconWide` on almost every icon and when `html !== false` you set `iconWide` again. That will result in double rendering of `fa-fw`

**Comment by N6REJ** [comment]
@hans2103 and I discussed this in the team and we're going to do it in another pr we're working on.

**Pull request closed without merging**